src/main/resources/i18n/messages_en.properties

```diff
@@ -90,3 +90,6 @@ delete-assessment.id.notNull='assessmentId' may not be empty
 delete-assessment.id.notFound='assessmentId' is not valid
 
 count-assessments.assessment.kit.id.notNull='assessmentKitId' may not be empty
+
+delete-evidence.evidence-not-found=evidence not found
```

Comment:
->
delete-evidence.id.notFound=no evidence found by this 'evidenceId'
delete-evidence.id.notNull='evidenceId' may not be empty